docs/topics/i18n/translation.txt

```diff
@@ -405,6 +407,15 @@ with the ``short_description`` attribute::
             verbose_name=_('kind'),
         )
 
+        @admin.display(description=_('Is it a mouse?'))
+        def is_mouse(self):
+            return self.kind.type == MOUSE_TYPE
+
+Note that this is also applicable if setting the ``short_description``
+attribute on the method directly instead of using the decorator::
+
+        ...
```

Comment:
I think this example would be clearer by showing the container:

```suggestion
    class MyThing(models.Model):
        ...
```

But I also wonder if it's still required - the full docs for `@display` show the equivalent direct attribute setting, so it seems redundant to re-mention it here.

Reply:
Yes. I wasn't originally sure, but I think we can drop it.